CriteoPublisherSdk/Sources/Configuration/CR_DeviceInfo.h

```diff
@@ -33,11 +33,16 @@ NS_ASSUME_NONNULL_BEGIN
 @property(copy, atomic) NSString *userAgent;
 @property(copy, nonatomic, readonly) NSString *deviceId;
 
+/**
+ * Full size of the main screen without considering the safe area.
+ */
+@property(nonatomic, readonly) CGSize screenSize;
+
 - (instancetype)initWithThreadManager:(CR_ThreadManager *)threadManager NS_DESIGNATED_INITIALIZER;
 
 - (void)waitForUserAgent:(void (^_Nullable)(void))completion;
-+ (CGSize)getScreenSize;
-+ (BOOL)validScreenSize:(CGSize)size;
+
+- (BOOL)validScreenSize:(CGSize)size;
```

Comment:
Probably unrelated to this PR but this one does not look like properly named. Should be something like `isScreenSizeValid:` or `validateScreenSize:`

Reply:
I think this does not belong to this PR.